data-raw/R2R_TMH_habitat_inputs/cache_tmh_dataR.R

```diff
@@ -0,0 +1,306 @@
+    adj_factor = 1
+  } else {
+    adj_factor = (max_hab_acres - existing_acres) / existing_acres + 1
+  }
+  
+  new_hab_acres <- DSMhabitat::fr_spawn$biop_itp_2018_2019[ws , , ] * adj_factor
+  
+  r_to_r_tmh_fr_spawn[ws, , ] <- new_hab_acres 
+  
+}
+
+
+## inchannel habitat to both fry and juvenile habitat objects ---------------
+### set r_to_r_baseline_fr_juv and fry 
+r_to_r_tmh_fr_juv <- DSMhabitat::fr_juv$biop_itp_2018_2019
+r_to_r_tmh_fr_fry <- DSMhabitat::fr_fry$biop_itp_2018_2019
+
+for(i in 1:length(watersheds)) {
+  ws = watersheds[i]
+  habitat = "rear"
+  
+  max_hab_acres <- all_hab_data_long |> 
+    filter(watershed == ws & hab == habitat) |> 
+    filter(max_hab == "max_hab") |> 
+    pull(value)
+  
+  existing_acres_juv =   all_hab_data_long |> 
+    filter(watershed == ws & hab == habitat) |> 
+    filter(unit != "cfs") |>  
+    filter(is.na(max_hab)) |> 
+    filter(lifestage == "juv") |> 
+    pull(value)
+  
+  existing_acres_fry =   all_hab_data_long |> 
+    filter(watershed == ws & hab == habitat) |> 
+    filter(unit != "cfs") |>  
+    filter(is.na(max_hab)) |> 
+    filter(lifestage == "fry") |> 
+    pull(value)
+  
+  # Note: If the maximum theoretical habitat was less than the existing SIT habitat, 
+  # the theoretical maximum habitat value was used for baseline and model runs. 
+  adj_factor_juv = (max_hab_acres - existing_acres_juv) / existing_acres_juv + 1
+  adj_factor_fry = (max_hab_acres - existing_acres_fry) / existing_acres_fry + 1
+  
+  add_max_hab_juv <- DSMhabitat::fr_juv$biop_itp_2018_2019[ws , , ] * adj_factor_juv
+  add_max_hab_fry <- DSMhabitat::fr_fry$biop_itp_2018_2019[ws , , ] * adj_factor_fry
+  
+  r_to_r_tmh_fr_juv[ws, , ] <- add_max_hab_juv 
+  r_to_r_tmh_fr_fry[ws, , ] <- add_max_hab_fry 
+  
+}
+
+
+
+##floodplain: -------------------------------------------------------------
+r_to_r_tmh_fr_flood <- DSMhabitat::fr_fp$biop_itp_2018_2019
+
+for(i in 1:length(watersheds)) {
+  ws = watersheds[i]
+  habitat = "flood"
+  
+  max_hab_acres <- all_hab_data_long |> 
+    filter(watershed == ws & hab == habitat) |> 
+    filter(max_hab == "max_hab") |> 
+    pull(value)
+  
+  existing_acres =   all_hab_data_long |> 
+    filter(watershed == ws & hab == habitat) |> 
+    filter(unit != "cfs") |>  
+    filter(is.na(max_hab)) |> 
+    pull(value)
+  
+  # Note: If the maximum theoretical habitat was less than the existing SIT habitat, 
+  # the theoretical maximum habitat value was used for baseline and model runs. 
+  if(existing_acres == 0) {
+    adj_factor = 1
+  } else {
+    adj_factor = (max_hab_acres - existing_acres) / existing_acres + 1
+  }
+  
+  
+  new_hab_acres <- DSMhabitat::fr_fp$biop_itp_2018_2019[ws , , ] * adj_factor
+  
+  r_to_r_tmh_fr_flood[ws, , ] <- new_hab_acres 
+  
+}
+
+##delta: -------------------------------------------------------------------
+
+watersheds <- c('North Delta', 'South Delta')
+r_to_r_tmh_delta <- DSMhabitat::delta_habitat$r_to_r_baseline
```

Comment:
Do we want to scale the baseline or the original - the baseline already has a ton of project habitat added

Reply:
outdated